### PR TITLE
Update RPC-O to track stable queens

### PR DESCRIPTION
**Pull request by cloudnull**
Update RPC-O to track stable queens

This change updates rpco to use queens in the master branch and removes
our old artifact sources which were implemented for Pike.

Issue: [INTAKERPC-172](https://rpc-openstack.atlassian.net/browse/INTAKERPC-172)
Issue: [INTAKERPC-132](https://rpc-openstack.atlassian.net/browse/INTAKERPC-132)
Issue: [RO-2932](https://rpc-openstack.atlassian.net/browse/RO-2932)

Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

### CODE REVIEW

**Comment by prometheanfire** [comment]
recheck_CIT_xenial_no_artifacts_swift_deploy

**Comment by cloudnull** [comment]
recheck_CIT_xenial_no_artifacts_swift_deploy

**Comment by prometheanfire** [comment]
```
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable.

The error was: 'rpco_mirror_base_url' is undefined

The error appears to have been in '/var/lib/jenkins/workspace/PR_rpc-openstack-master-xenial_no_artifacts-swift-deploy/rpc-openstack/playbooks/configure-apt-sources.yml':
line 63, column 7, but may be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - name: Check for artifacts
      ^ here\n\nexception type: <class 'ansible.errors.AnsibleUndefinedVariable'>
exception: 'rpco_mirror_base_url' is undefined"}
```

**Comment by odyssey4me** [comment]
The artifact job would have to understand how to cater for these changes. If the same repo/branch combination is being used for Pike & Queens then that may get complicated. This tells me that the artifact implementation is slowing us down and we should remove it until we have a better implementation that is upstream first and carries as little as possible downstream.

**Comment by cloudnull** [comment]
recheck_cit_release

**Comment by prometheanfire** [comment]
Cannot contact promxnasd-90-76fe: java.io.IOException: remote file operation failed: /var/lib/jenkins/workspace/PR_rpc-openstack-master-xenial_no_artifacts-swift-deploy at hudson.remoting.Channel@15081d94:promxnasd-90-76fe: hudson.remoting.ChannelClosedException: channel is already closed

wat?

**Comment by cloudnull** [comment]
@prometheanfire fixed merge conflict.

File: gating/pre_merge_test/run_system_tests.sh [hunk 1]
```diff
@@ -30,14 +30,7 @@ pushd "${SYS_TEST_SOURCE}"
 git checkout "${SYS_TEST_BRANCH}"
 echo "${SYS_TEST_SOURCE} at SHA $(git rev-parse HEAD)"
 
-# Gather submodules
```
Comment (odyssey4me) [inline gating/pre_merge_test/run_system_tests.sh]: I don't think these changes should be included. The submodule here is not one in RPC-O, but in the system tests repo.

**Pull request merged**